### DISCUSSION TITLE
Fundamentos do ASP.NET 6 > CRUD e Entity Framework > Padronizando erros

### DISCUSSION THREAD

**Discussion by weslleyrlima** [balta-io/forum#419] [Tira Dúvidas] [answered]
Boa tarde Balta,

Sobre a questão da padronização de códigos no log que você deu como exemplo "05X04 - Falha interna no servidor", em um sistema muito maior como você controlaria os códigos para não haver repetição? Minha idéia seria adicionar os códigos em uma tabela somente para suporte mesmo, com keys únicas. Qual sua opnião?

**Comment by andrebaltieri** [comment, marked as answer]
@weslleyrlima bom dia!
No caso, este controle a empresa faz como acreditar ser melhor. Pode ser um Excel, um Knowledge Base, uma documentação e até uma tabela como citou!

**Reply by weslleyrlima** [reply]
Bom dia @andrebaltieri  ! Ok obrigado. 
Quanto ao padrão do código de log (exp 05X04), você usa algum padrão em específico nos seus projetos ou tb a empresa que define qual o melhor?

**Reply by andrebaltieri** [reply]
Normalmente uso os primeiros 8 dígitos de um GUID! Mas não é uma regra, só uma sugestão.

**Reply by weslleyrlima** [reply]
É aquela coisa do KISS né "Keep it simple, stupid!"... tem que ser simples e objetivo. Vlw Balta!
